backend/db/migrate/20170801101708_create_clients.rb

```diff
@@ -0,0 +1,9 @@
+class CreateClients < ActiveRecord::Migration[5.1]
+  def change
+    create_table :clients do |t|
+      t.string :token
+
```

Comment:
I see, I have [added `name` on clients table](https://github.com/ninariccimarie/quiz-master/pull/6/commits/620106835fb0383c4200a02627a3f7967b5ce6ca) for this.